packages/react/package.json

```diff
@@ -39,6 +39,7 @@
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-test-renderer": "^16.13.1",
+    "redux": "^4.0.5",
```

Comment:
As redux is only used for typing I don't think we should have redux as a `peerDependency` because it would just add an extra unneeded dependency for users who don't use the redux integration in their react apps.

Reply:
Then can we make it an `optionalDependency`?

Reply:
Hmm sounds good to me.

Reply:
By having redux as an `optionalDependency` we now get a `no-implicit-dependencies` warning from tslint though, should we just ignore that?